insights/specs/insights_archive.py

```diff
@@ -178,6 +178,8 @@ class InsightsArchiveSpecs(Specs):
     oc_get_route = simple_file("insights_commands/oc_get_route_-o_yaml_--all-namespaces")
     oc_get_service = simple_file("insights_commands/oc_get_service_-o_yaml_--all-namespaces")
     oc_get_configmap = simple_file("insights_commands/oc_get_configmap_-o_yaml_--all-namespaces")
+    openshift_fluentd_pid = simple_file("insights_commands/pgrep_-n_fluentd")
```

Comment:
For `foreach_collect`, it's not necessary to be added to the `insights_archive.py` for now.